playbooks/common/openshift-checks/health.yml

```diff
@@ -1,8 +1,5 @@
 ---
-# openshift_health_checker depends on openshift_version which now requires group eval.
 - include: ../openshift-cluster/evaluate_groups.yml
-  tags:
-  - always
```

Comment:
If `--tags` is passed in the command line, it makes no sense to evaluate groups and then don't run anything else. AFAICT we don't really need tags.